src/Indexer.js

```diff
@@ -81,19 +126,41 @@ export default class Indexer {
    */
   identifierFrom(uri) {
     // pathname looks like /baz-quux-quuux; remove the slash
-    const identifier = new Url(uri).pathname.substr(1) || Config.rootNodeIdentifier
+    const identifier = new Url(uri).pathname.substr(1) || config.get('rootNodeIdentifier')
     this.logger.debug(`identifier from ${uri} is ${identifier}`)
     return identifier
   }
 
+  /**
+   * Builds up an index entry out of a JSON body, given index field mappings from config
+   * @param {Object} json - A Trellis resource (of some kind: RDFSource, BasicContainer, NonRDFSource, etc.)
+   * @returns {Object} an object containing configured field values if any found
+   */
+  buildIndexEntryFrom(json) {
```

Comment:
I like this name.  In ruby it would probably be `indexEntry`

Reply:
I was intentional about using `build` here since the method is not indexing an entry (that's what `index()` does; rather, it was building an index entry (or index document) from given json. Glad you like the name! I'm kinda on the fence about the `From` part, tbh, but `/shrug`. :smile:

Reply:
i like the `From` part 🙂 

agreed about the disambiguating prefix of `build`, that's nice.